### PR TITLE
[zh-cn]: Update translation of z-index

### PR DESCRIPTION
**Pull request by jynxio**
## Description

Update the Simplified Chinese translation of `z-index`.

## Motivation

The translated section on "取值" in the original translation contains errors and is completely incomprehensible.

## Additional details

- Update the section on "取值".
- Change "堆叠上下文" to "层叠上下文" (referenced from [here](https://developer.mozilla.org/zh-CN/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context)).
- Make other minor updates.

----

## 描述

更新 `z-index` 的简中翻译。

## 动机

原始翻译的 “取值” 小节存在翻译错误且根本无法看懂。

## 细节

- 更新 “取值” 小节；
- 将 “堆叠上下文” 改为 “层叠上下文”（参考自 [此处](https://developer.mozilla.org/zh-CN/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context)）；
- 其它的微小更新；

### CODE REVIEW

File: files/zh-cn/web/css/z-index/index.md [hunk 1]
```diff
@@ -5,19 +5,19 @@ slug: Web/CSS/z-index
 
 {{CSSRef}}
 
-`z-index` 属性设定了一个定位元素及其后代元素或 flex 项目的 z-order。当元素之间重叠的时候，z-index 较大的元素会覆盖较小的元素在上层进行显示。
+`z-index` 属性设置定位元素及其后代元素或 flex 项目的 Z 轴顺序，在发生重叠的元素之间，`z-index` 的较大者会覆盖较小者。
```
Comment (yin1999) [inline files/zh-cn/web/css/z-index/index.md:8]:
```suggestion
CSS **`z-index`** 属性设置定位元素及其后代元素或 flex 项目的 Z 轴顺序。z-index 的较大的重叠元素会覆盖较小的元素。
```

File: files/zh-cn/web/css/z-index/index.md [hunk 2]
```diff
@@ -5,19 +5,19 @@ slug: Web/CSS/z-index
...
 
 {{EmbedInteractiveExample("pages/css/z-index.html")}}
 
-对于一个已经定位的盒子（即其 `position` 属性值不是 `static`，这里要注意的是 CSS 把元素看作盒子），`z-index` 属性指定：
+对于定位元素（即 `position` 属性值非 `static` 的元素），`z-index` 属性会指定：
```
Comment (yin1999) [inline files/zh-cn/web/css/z-index/index.md:12]:
```suggestion
对于定位盒子（即 `position` 属性值非 `static` 的盒子），`z-index` 属性会指定：
```

File: files/zh-cn/web/css/z-index/index.md [hunk 3]
```diff
@@ -5,19 +5,19 @@ slug: Web/CSS/z-index
...
 
-1. 盒子在当前堆叠上下文中的堆叠层级。
-2. 盒子是否创建一个本地堆叠上下文。
+1. 盒子在当前 [层叠上下文](/zh-CN/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context) 中的层叠等级。
```
Comment (yin1999) [inline files/zh-cn/web/css/z-index/index.md:14]:
```suggestion
1. 盒子在当前[层叠上下文](/zh-CN/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context)中的层叠等级。
```

**Review by yin1999 — COMMENTED** [review]
Thanks @jynxio. Just some suggestions

**Comment by jynxio** [comment]
Thanks @yin1999 

I agree with your suggestion, but I didn't know how to automatically add your proposed content in the PR, so I ended up using the most basic method (manual editing).

Additionally, I made a very small change where I replaced 「z-index 的较大的重叠元素会覆盖较小的元素」 with 「z-index 较大的重叠元素会覆盖较小的元素」 because I believe the 「的」 is redundant in terms of grammar.

**Comment by yin1999** [comment]
> I agree with your suggestion, but I didn't know how to automatically add your proposed content in the PR, so I ended up using the most basic method (manual editing).

https://docs.github.com/zh/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request

File: files/zh-cn/web/css/z-index/index.md [hunk 4]
```diff
@@ -107,4 +113,4 @@ z-index: unset;
 ## 参阅
 
 - CSS {{Cssxref("position")}} 属性
-- [理解 CSS z-indexes](/zh-CN/docs/CSS/Understanding_z-index)
+- [理解 CSS z-index](Web/CSS/CSS_positioned_layout/Understanding_z-index)
```
Comment (yin1999) [inline files/zh-cn/web/css/z-index/index.md:116]:
```suggestion
- [理解 CSS z-index](/zh-CN/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index)
```

**Review by yin1999 — APPROVED** [review]
Thanks @jynxio. I've allowed myself to do a full sync fot this document :)

**Review by yin1999 — APPROVED** [review]
Thanks @jynxio. I've allowed myself to do a full sync fot this document :)

**Pull request merged**